zone/CMakeLists.txt

```diff
@@ -294,6 +294,7 @@ SET(gm_commands
     gm_commands/aggrozone.cpp
     gm_commands/ai.cpp
     gm_commands/appearance.cpp
+	gm_commands/appearanceeffects.cpp
```

Comment:
Formatting. File uses 4 spaces for some reason.

Reply:
resolved